test/acceptance/behave/components/ngsiv2/entities/create_entities.feature

```diff
@@ -0,0 +1,1889 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+# author: 'Iván Arias León (ivan dot ariasleon at telefonica dot com)'
+
+
+Feature: create entities requests (POST) in NGSI v2
+  As a context broker user
+  I would like to  create entities requests in NGSI v2
+  So that I can manage and use them in my scripts
+
+  BackgroundFeature:
+  Setup: update properties test file from "epg_contextBroker.txt" and sudo local "false"
+  Setup: update contextBroker config file and restart service
+  Check: verify contextBroker is installed successfully
+  Check: verify mongo is installed successfully
+
+  @happy_path
+  Scenario:  create a new entity in NGSI v2
+    Given  a definition of headers
+      | parameter          | value            |
+      | Fiware-Service     | test_happy_path  |
+      | Fiware-ServicePath | /test            |
+      | Content-Type       | application/json |
+    When create "1" entities with "1" attributes
```

Comment:
why you use "1" entities and "1" attribute if you are describing a bunch of them?